Cargo.toml

```diff
@@ -26,12 +26,14 @@ sha2 = "^0.10"
 thiserror = "^1.0"
 uuid = { version = "^1.0", features = ["serde"] }
 once_cell = "^1.8"
+uds = "^0.2"
 
 [dev-dependencies]
 quickcheck = "^1.0"
 serde_json = "^1.0"
 rand = "^0.8"
 pretty_assertions = "^1.0"
+memfd = "^0.6"
```

Comment:
I can change it to use a regular File instead. The end result should be the same.

Reply:
memfd seem to be using something called "rustix" instead of nix. So maybe it will bring a bit of extra heft.

It's just an example so creating a file and unlinking it immediatly will demostrate how to use it as well.

Reply:
Full disclosure: I do maintain the `memfd` crate.

I think longer term it would make sense to start using `rustix` and benefit from the new `io-safety` primitives. So this `memfd` dev-dependency may end up here anyway.

But for the moment we can try to avoid that. Recent `nix` should have basic memfd support.

Reply:
Yes, possibly. Or at least properly using `BorrowedFd`/`OwnedFd`. But `io-safety` is quite recent and the whole topic still evolving. Let's stick to plain `nix` for the moment and avoid this `memfd` dependency for now.